typeform-helpers.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const request = require('request'),
+    _ = require('lodash'),
+    moment = require('moment'),
+    TYPEFORM_TOKEN = require('./api-keys').TYPEFORM_TOKEN,
+    constants = require('./constants'),
+    TYPEFORM_FORM_ID = constants.TYPEFORM_FORM_ID,
+    TYPEFORM_EMAIL_FIELD = constants.TYPEFORM_EMAIL_FIELD,
+    TYPEFORM_TWITTER_FIELD = constants.TYPEFORM_TWITTER_FIELD
+
+function getFromTypeform (params, callback) {
```

Comment:
A generic function to get responses from typeform API. The user can pass in params like `offset`, `since`, `completed` etc. If no params are passed in, this function will return all responses from the API.